src/main/java/org/randywebb/wlts/util/AppConfig.java

```diff
@@ -13,18 +13,21 @@
 * @author randyw
 *
 */
-public class AppConfig extends Properties {
+public final class AppConfig extends Properties {
```

Comment:
Not sure why this needs to be final. I guess it's OK, but I don't particularly need to restrict anyone from extending it.
What am I missing?

Reply:
I think it was because it follows the Singleton model (if I recall correctly). If we need to refactor for subclasses we can remove the final as part of the refactor.

Reply:
Cool! I've learned something :)
I can't tell you how many singletons I've done without making the class final, but I can learn better!